src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/EntityDtoValidatorTest.java

```diff
@@ -39,37 +44,227 @@ void testValidateNameIsEmpty() {
         entityDto.setName("  ");
         Errors errors = entityDtoValidator.validate(entityDto, new Errors(), CONTEXT);
 
-        assertError(EntityDto.NAME_FIELD, ValidationMessages.NOT_EMPTY_ERROR_MESSAGE, errors);
+        assertError(EntityDto.NAME_FIELD, ValidationMessages.NOT_EMPTY_ERROR_MESSAGE, errors, true);
     }
 
     @Test
     void testValidateNameIsNull() {
         entityDto.setName(null);
         Errors errors = entityDtoValidator.validate(entityDto, new Errors(), CONTEXT);
 
-        assertError(EntityDto.NAME_FIELD, ValidationMessages.NOT_NULL_ERROR_MESSAGE, errors);
+        assertError(EntityDto.NAME_FIELD, ValidationMessages.NOT_NULL_ERROR_MESSAGE, errors, true);
     }
 
     @Test
     void testValidateNameLength() {
         entityDto.setName(StringUtils.repeat("A", 161));
         Errors errors = entityDtoValidator.validate(entityDto, new Errors(), CONTEXT);
 
-        assertError(EntityDto.NAME_FIELD, " must be 160 characters or less", errors);
+        assertError(EntityDto.NAME_FIELD, " must be 160 characters or less", errors, true);
     }
 
     @Test
     void testValidateNameCharacters() {
         entityDto.setName("Дракон");
         Errors errors = entityDtoValidator.validate(entityDto, new Errors(), CONTEXT);
 
-        assertError(EntityDto.NAME_FIELD, ValidationMessages.INVALID_CHARACTERS_ERROR_MESSAGE, errors);
+        assertError(EntityDto.NAME_FIELD, ValidationMessages.INVALID_CHARACTERS_ERROR_MESSAGE, errors, true);
+    }
+
```

Comment:
Same here - test with multiple failures is missing.

Reply:
Added this test.